modules/primer-popover/lib/popover.scss

```diff
@@ -108,7 +108,7 @@
   }
 
   &::before {
-    margin-top: -9px;
+    margin-top: -($spacer-2 + 1);
```

Comment:
I know it's not a result of this PR, but it seems like this `margin-top` makes it so that the distance from the caret to the corner of the card is smaller for --right and --left popovers as compared to --top and --bottom popovers. Is this intended? Or should we have consistent distances from the carets to the corners of the popovers?

Smaller distance to corner:
<img width="37" alt="screen shot 2018-03-27 at 11 27 40 am" src="https://user-images.githubusercontent.com/180798/37977587-0722471c-31b2-11e8-8fcc-24b6978a75dd.png">
Larger distance to corner:
<img width="77" alt="screen shot 2018-03-27 at 11 27 19 am" src="https://user-images.githubusercontent.com/180798/37977591-08bf2392-31b2-11e8-8c25-33fc8ab28cc9.png">

We have some JS that tries to position the popovers relative to the hover target in a generic way, and it would be convenient if the distance from the caret to the corner was always consistent 😄. Thoughts?

Reply:
> For what it's worth, though, this value actually hasn't changed; I'm just calculating it as a nudge from a "known" value ($spacer-2: 8px) rather than hard-coding 9px.

Right on - was asking an unrelated question 😄 

Bummer that the distances aren't consistent, but we can handle it.

Thanks for taking this on!

Reply:
Haha, I tried that at one point! We'll consider that, especially if we run into more issues with our current approach

Reply:
I'm not sure of the reasons for the distances being different. @brandonrosage worked on adding this component but likely followed the custom styles that existed in dotcom. @pifafu wondering if you have any opinion on this with regards to using on hovercards?

Reply:
>wondering if you have any opinion on this with regards to using on hovercards?

I didn't see a need for the distances between the corner of the card and the caret being different when the caret is top/bottom versus right/left for the user hovercard project. If the caret distance-to-corner was made consistent, we would be good with it  👍

The only spacing that the hovercards project wanted to update on was the distance of the hovercard from the entity that triggers it, since the original styles on the popover were flush to the triggering element.

![screen shot 2018-04-11 at 1 12 32 pm](https://user-images.githubusercontent.com/5679180/38640763-083bf498-3d8a-11e8-8f85-caa7c339f55e.png)

Reply:
@shawnbot our JS code positions the hovercard so that it's 12 pixels from the target element

Reply:
>Have you all done that in the implementation (e.g. with margin utility classes), or should we account for it in the component styles?

@shawnbot I [started with](https://github.com/github/github/pull/85741/files) `pb-2` to give more space when we started with the popover component, but @iancanderson's recent changes now help handle this! @califa also has interest in larger spacing between the triggering element on the popover component (`mt-3` is too large, `mt-2` is not enough?). It would be nice to have this accounted for in component styles 😄